client/changelist/change.go

```diff
@@ -58,8 +58,8 @@ func (c TUFChange) Action() string {
 }
 
 // Scope returns c.Role
-func (c TUFChange) Scope() string {
-	return c.Role
+func (c TUFChange) Scope() data.RoleName {
+	return data.RoleName(c.Role)
```

Comment:
`c.Role` should already have the type `data.RoleName` per the struct definition.